setup.py

```diff
@@ -18,7 +18,6 @@
 ]
 
 DEPENDENCY_LINKS = [
-    'git+https://github.com/monzo/pagerduty-api-python-client.git#egg=pypd',
 ]
```

Comment:
We can remove this entirely - I think we should avoid using it if at all possible